### PR TITLE
Quagga bgp peer work

### PR DESCRIPTION
**Pull request by m4ce**
* Some cosmetic changes..
* Created quagga::bgp::peer define type

### CODE REVIEW

**Comment by m4ce** [comment]
There are probably some mistakes.. have not actually tested it! :/

**Comment by bibigon812** [comment]
I'll merge it late and fix tests.

File: manifests/bgp/peer.pp [hunk 1]
```diff
@@ -0,0 +1,32 @@
+define quagga::bgp::peer (
+  Variant[Enum["absent"], Integer[1, 4294967295]] $local_as,
+  Variant[Enum["absent"], Integer[1, 4294967295]] $remote_as,
```
Comment (bibigon812) [inline manifests/bgp/peer.pp]: Optional[Integer[1, 4294967295]] $local_as,
Optional[Integer[1, 4294967295]] $remote_as,

File: manifests/bgp/peer.pp [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  Boolean $passive,
```
Comment (bibigon812) [inline manifests/bgp/peer.pp]: Optional[Boolean] $passive,

**Comment by bibigon812** [comment]
@m4ce Fixed.

**Pull request merged**